scripts/qa_crowbarsetup.sh

```diff
@@ -622,6 +622,8 @@ function onadmin_prepare_cloud_repos()
             develcloud5)
                 addsp3testupdates
                 add_sles12ga_testupdates
+                addcloud5maintupdates
```

Comment:
Could that cause problems when there are version with higher build numbers than in D:C:5:S?